.ibm/pipelines/auth/secrets-rhdh-secrets.yaml

```diff
@@ -14,7 +14,8 @@ data:
   GITHUB_ORG: amFudXMtcWU=
   K8S_CLUSTER_NAME: bXktY2x1c3Rlcgo=
   K8S_CLUSTER_API_SERVER_URL: aHR0cHM6Ly9jMTAwLWUudXMtZWFzdC5jb250YWluZXJzLmNsb3VkLmlibS5jb206MzIyMTIK
-  K8S_SERVICE_ACCOUNT_TOKEN: dGVtcA==
+  K8S_CLUSTER_TOKEN: dGVtcA==
```

Comment:
@gustavolira do we need to have `K8S_CLUSTER_TOKEN` in this file? if not, can we remove it? I noticed that `K8S_CLUSTER_TOKEN` is used with 'oc' command in `.ibm/pipelines/openshift-ci-tests.sh` but the real value is set by `.ibm/pipelines/env_variables.sh`